jsscripts/UserAgentOverrides.jsm

```diff
@@ -0,0 +1,173 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
Please add sha1 information from the github gecko-mirror so that it's easier to track and check. I have missed some but have since added this kind of comments:

// Ported from Android FF esr60 sha1 c714053d73ac408ab402bb4d7e906e718f4ecb7e

I think github gecko-mirror link would be even better and easier. 



This got removed in https://github.com/mozilla/gecko-dev/commit/b27c4726d099aa7f49c26f4c27955d49a228ba5b and I'm assuming that this is the version just before that.

It might well be that this is not yet working like this but this is a good first step. Could please log a task to verify that user agent overrides are working?